consensus/spos/worker.go

```diff
@@ -200,7 +203,7 @@ func (wrk *Worker) receivedSyncState(isNodeSynchronized bool) {
 	}
 }
 
-func (wrk *Worker) ReceivedHeader(headerHandler data.HeaderHandler, headerHash []byte) {
+func (wrk *Worker) ReceivedHeader(headerHandler data.HeaderHandler, _ []byte) {
 	if headerHandler.GetShardID() != wrk.shardCoordinator.SelfId() ||
```

Comment:
maybe you can create a bool variable for these 3 conditions

Reply:
agree, done